### ISSUE TITLE
[bug] crashed on terminal while trying to play an unselected card

### ISSUE THREAD

**Issue by kaandesu** [BigJk/end_of_eden#1]
Hello, I've seen this repo super recently downloaded the executable for the first time from the releases at
[end_of_eden_term-v0.1.10-macos-amd64.zip](https://github.com/BigJk/end_of_eden/releases/download/v0.1.10/end_of_eden_term-v0.1.10-macos-amd64.zip)
while playing I noticed it crashed a few times. I am on Mac14 + iTerm2.

I was just instinctively using 'enter'  to select the cards and click the end turn by mouse. This error seem to happen when the card is not selected as default, so when i click enter it logged"runtime error: index out of range [-1]". full log below:

https://github.com/BigJk/end_of_eden/assets/74111241/d778b8e7-d13b-4e53-a0e9-7f72070098a5

```bash
End Of Eden
Initializing Settings. Please wait...
Done!
Initializing Audio. Please wait...
Done!
Initializing Proc-Gen. Please wait...
Done!
Initializing Localization. Please wait...
Done!

Caught panic:

runtime error: index out of range [-1]

Restoring terminal...

goroutine 1 [running]:
runtime/debug.Stack()
	/Users/runner/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.21.6.darwin-amd64/src/runtime/debug/stack.go:24 +0x5e
runtime/debug.PrintStack()
	/Users/runner/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.21.6.darwin-amd64/src/runtime/debug/stack.go:16 +0x13
github.com/charmbracelet/bubbletea.(*Program).Run.func1()
	/Users/runner/go/pkg/mod/github.com/!big!jk/bubbletea@v0.0.0-20231222122351-25c0e0b15c34/tea.go:478 +0x91
panic({0x10110cd00?, 0xc004b90108?})
	/Users/runner/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.21.6.darwin-amd64/src/runtime/panic.go:920 +0x270
github.com/BigJk/end_of_eden/ui/menus/gameview.Model.tryCast({{{0xc8, 0x33}, {0xb5, 0x31, 0x0, 0x0, 0x0, 0x2, 0x0, 0xb}, ...}, ...})
	/Users/runner/work/end_of_eden/end_of_eden/ui/menus/gameview/gameview.go:431 +0x4b2
github.com/BigJk/end_of_eden/ui/menus/gameview.Model.Update({{{0xc8, 0x33}, {0xb5, 0x31, 0x0, 0x0, 0x0, 0x2, 0x0, 0xb}, ...}, ...}, ...)
	/Users/runner/work/end_of_eden/end_of_eden/ui/menus/gameview/gameview.go:100 +0x479
github.com/BigJk/end_of_eden/ui/root.Model.Update({0xc000440310, {0xc00045a240, 0x2, 0x4}, {0xc8, 0x33}, 0xc002619740, 0x0}, {0x1010e8a80, 0xc002618030})
	/Users/runner/work/end_of_eden/end_of_eden/ui/root/root.go:86 +0x5d8
github.com/charmbracelet/bubbletea.(*Program).eventLoop(0xc00036d930, {0x101bcde18?, 0xc00041c040?}, 0xc00146dc70?)
	/Users/runner/go/pkg/mod/github.com/!big!jk/bubbletea@v0.0.0-20231222122351-25c0e0b15c34/tea.go:411 +0x75a
github.com/charmbracelet/bubbletea.(*Program).Run(0xc00036d930)
	/Users/runner/go/pkg/mod/github.com/!big!jk/bubbletea@v0.0.0-20231222122351-25c0e0b15c34/tea.go:543 +0x86e
main.main()
	/Users/runner/work/end_of_eden/end_of_eden/cmd/game/main.go:138 +0x1465
```

**Comment by BigJk** [comment]
Hey! Thanks for reporting this crash. It should be fixed with v0.1.11. Feel free to check and report back 😀

**Comment by kaandesu** [comment]
Hi,i am happy to confirm it is fixed, thanks for the reactivity! also as a fan of slay the spire and go with bubbletea i really appreciate the project super cool!
